### PR TITLE
Can be process short array in row

### PR DESCRIPTION
**Pull request by bscheshirwork**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | yes/no
| Tests pass?   | yes/no
| Fixed issues  | #14697

### CODE REVIEW

**Comment by bscheshirwork** [comment]
@samdark done!

**Comment by samdark** [comment]
Tests are failing.

**Comment by bscheshirwork** [comment]
@samdark 
Fixed! :smile_cat:

**Comment by bscheshirwork** [comment]
@samdark  
well :smiley_cat: "All checks have passed" fixed!

**Comment by samdark** [comment]
Merged. Thanks!

File: framework/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ Yii Framework 2 Change Log
 2.0.13 under development
 ------------------------
 
+- Bug #14697: Fixed `console\widgets\Table` rendering wherne there's no data supplied (bscheshirwork)
```
Comment (InsaneSkull) [inline framework/CHANGELOG.md]: `wherne` to `where`?

**Pull request merged**